packages/core/strapi/lib/commands/opt-out-telemetry.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const { resolve } = require('path');
+const fse = require('fs-extra');
+const chalk = require('chalk');
+const fetch = require('node-fetch');
+const machineID = require('../utils/machine-id');
+
+const readPackageJSON = async path => {
+  try {
+    const packageObj = await fse.readJson(path);
+    const uuid = packageObj.strapi.uuid;
+
+    return [uuid, packageObj];
+  } catch (err) {
+    console.error(`${chalk.red('Error')}: ${err.message}`);
+  }
+};
+
+const writePackageJSON = async (path, file, spacing) => {
+  try {
+    await fse.writeJson(path, file, { spaces: spacing });
+    return true;
+  } catch (err) {
+    console.error(`${chalk.red('Error')}: ${err.message}`);
+  }
+};
+
+const sendEvent = async uuid => {
+  try {
+    await fetch('https://analytics.strapi.io/track', {
+      method: 'POST',
+      body: JSON.stringify({
+        event: 'didOptOutTelemetry',
+        uuid,
+        deviceId: machineID(),
+      }),
+      headers: { 'Content-Type': 'application/json' },
+    });
+  } catch (e) {
+    //...
+  }
+};
+
+module.exports = async function optOutTelemetry() {
+  const packageJSON = resolve(process.cwd(), 'package.json');
+
+  if (!packageJSON) {
+    console.log(`${chalk.yellow('Warning')}: could not find package.json`);
+    process.exit(0);
+  }
+
+  const [uuid, packageObj] = await readPackageJSON(packageJSON);
+
+  if (packageObj.strapi.optOutTelemetry || !uuid) {
+    console.log(`${chalk.yellow('Warning:')} telemetry is already disabled`);
+    process.exit(0);
+  }
+
+  const updatedPackageJSON = {
+    ...packageObj,
+    strapi: {
+      uuid: packageObj.strapi.uuid,
+      optOutTelemetry: true,
```

Comment:
Maybe it's not important but should we call this property `telemetryDisabled` to keep it in line with the existing environment variable naming? It's also a bit more straightforward naming I feel.

Not asking to rename the event name we send, just this `package.json` property :)

Reply:
I like these suggestions, thank you. Have made changes 👍